packages/vite-plugin-kit-routes/src/lib/plugin.ts

```diff
@@ -234,6 +234,7 @@ export type ExtendParam = {
 
 export type ExplicitSearchParam = ExtendParam & {
   required?: boolean
+  joinArray?: boolean
```

Comment:
There are 2 modes: 'join' | `split`

`join` => ids=1,2,3
`split` => ids=1&ids=2&ids=3

Right?

What do you think about doing like https://github.com/jycouet/kitql/blob/main/packages/vite-plugin-kit-routes/src/lib/plugin.ts#L146 ? (I would put `join` as default default. 

Let me know

Reply:
Nice idea, will change it.